### ISSUE TITLE
Guests can brake blocks with Toolbelt in protected regions.

### ISSUE THREAD

**Issue by ayyy** [TVPT/VoxelGuest#64]
Greetings.

Not sure if this is old news, as I believe there was some region issues after OSD. Anyway, the case is that it seems that guests can use any Toolbelt item to brake blocks in regions, such as Facet and Holocene. Also, apologize for multiple bugs on one thingie magicky. But /afk is broken at least when it does a random message. It instantly makes you return.

VoxelGuest-5.0.0-snapshot-jnks71
(all teh stuff on tvb)

**Comment by Peter200lx** [comment]
Please don't have two bugs in one issue, open separate bugs for unrelated issues.

As far as ToolBelt tools not respecting region protection, if that is the case then there is a bug. There are three possibilities, bug in ToolBelt, bug in VoxelGuest, or most likely something is mis-configured. I've personally tested ToolBelt in different regions on the Voxel server in the past, and made sure that it worked. I've not made any changes to ToolBelt since then that should have re-introduced the issue. If VoxelGuest is blocking you from breaking blocks by hand, then ToolBelt should be blocked as well. Thus the most likely case is that something in a configuration file has changed.

Normally I'd ask you to provide the config.yml file, but it sounds like you are reporting a problem related to a specific server, and not one you have control over. If this is the case, then you should be trying to contact a mod or admin on the server in question to have them try and investigate the issue. I can't solve problems when the exact configuration is unknown. I'll try and contact the Voxel folks later this week when I have time to sort this out with them.

So that I can be productive when talking with them, please answer the following questions:
What regions specifically are you having the problems in?

In those regions, can you break blocks by hand?

In those regions, can you place blocks by hand?

In those regions, can you use the jackhammer, sledge and paint tools?

What permission level are you, or the people seeing this issue?

**Comment by nristock** [comment]
Meh is guess he's actually talking about TVB :P  ... We didn't have a region system in Guest for OSD (and a few days after that). Butters and I made a very simple plugin to prevent the guests from building everywhere except the guest area. However, we implemented the region module now. So, the problems I can imagine are:

a) Someone removed the temporary plugin and didn't enable the region system in Guest
b) Someone enabled the region system in Guest but didn't migrate the old regions to the new persistence system
c) Someone messed up the coordinates again
d) VoxelGuest felt like breaking

...Since the Guest region protection was working last time I checked and nobody changed the code since then, it is most likely - as Peter200 mentioned - a configuration issue.
